osu.Game/Localisation/BeatmapSubmissionStrings.cs

```diff
@@ -39,6 +39,31 @@ public static class BeatmapSubmissionStrings
         /// </summary>
         public static LocalisableString SubmissionSettings => new TranslatableString(getKey(@"submission_settings"), @"Submission settings");
 
+        /// <summary>
+        /// "Submit beatmap!"
+        /// </summary>
+        public static LocalisableString ConfirmSubmission => new TranslatableString(getKey(@"confirm_submission"), @"Submit beatmap!");
+
+        /// <summary>
+        /// "Exporting beatmap set in compatibility mode..."
+        /// </summary>
+        public static LocalisableString ExportingBeatmapSet => new TranslatableString(getKey(@"exporting_beatmap_set"), @"Exporting beatmap set in compatibility mode...");
+
+        /// <summary>
+        /// "Preparing beatmap set online..."
+        /// </summary>
+        public static LocalisableString PreparingBeatmapSet => new TranslatableString(getKey(@"preparing_beatmap_set"), @"Preparing beatmap set online...");
+
+        /// <summary>
+        /// "Uploading beatmap set contents..."
+        /// </summary>
+        public static LocalisableString UploadingBeatmapSetContents => new TranslatableString(getKey(@"uploading_beatmap_set_contents"), @"Uploading beatmap set contents...");
```

Comment:
Any reason these strings have "set" and not the others? The term "beatmap set" only appears in code comments/logs and I believe just "beatmap" works fine here.

If intentional, going by the wiki, it should be "beatmapset" as stated "sometimes called" in https://osu.ppy.sh/wiki/en/Beatmap.

Reply:
https://github.com/ppy/osu/pull/31805/commits/753eae426d7c33978621025424b8dd43081a31fc